packages/ingest-mongodb-public/src/PublicIngestEnvVars.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  CORE_CHATBOT_APP_ENV_VARS,
+  CORE_OPENAI_EMBEDDING_ENV_VARS,
+} from "mongodb-rag-core";
+import { INGEST_MDBU_ENV_VARS } from "mongodb-rag-ingest";
+
```

Comment:
also need dev center DB i think

Reply:
Updated the import to use `INGEST_ENV_VARS` (which includes dev center and MDBU) instead of just `INGEST_MDBU_ENV_VARS`